docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md

```diff
@@ -148,6 +148,121 @@ In the default OMS configuration, seven days are allocated to Stripe to complete
 
 For instructions on using payment details, like the payment reference, from Stripe, see [Retrieve and use payment details from third-party PSPs](https://docs.spryker.com/docs/pbc/all/payment-service-provider/{{page.version}}/base-shop/retrieve-and-use-payment-details-from-third-party-psps.html)
 
+## Embed the Stripe payment page using iframe
+
+By default, the Stripe App payment flow assumes that the payment page is on another domain. When a user places an order, they're redirected to the Stripe payment page. To improve the user experience, you can embed the Stripe payment page directly into your website as follows:
+
+
+1. Create or update `PaymentConfig.php` (`Pyz\Zed\Payment\PaymentConfig`) on the project level with the following configuration:
```

Comment:
Can it be `Pyz\Zed\Payment\PaymentConfig\PaymentConfig.php`?

Reply:
yes, in this case `src/Pyz/Zed/Payment/PaymentConfig.php`
```suggestion
1. Create or update `src/Pyz/Zed/Payment/PaymentConfig.php` with the following configuration:
```